netweaver/install_aas.sls

```diff
@@ -28,6 +29,16 @@ create_aas_inifile_{{ instance_name }}:
         hana_password: {{ netweaver.hana.password }}
         hana_inst: {{ hana_instance }}
 
+{% if node.extra_parameters is defined %}
+update_aas_inifile_{{ instance_name }}:
+  module.run:
+    - netweaver.update_conf_file:
+      - conf_file: {{ inifile }}
+      - {%- for key,value in node.extra_parameters.items() %}
```

Comment:
perhaps we could rename `key` to `option`. Option/value sound less technical and intuitive then a generic key/value, since we are adding configuration option.